drivers/i2s/i2s_ll_stm32.h

```diff
@@ -58,4 +59,16 @@ struct i2s_stm32_data {
 	struct stream tx;
 };
 
+/* checks that DMA Tx packet is fully transmitted over the I2S */
+static inline uint32_t ll_func_i2s_dma_busy(SPI_TypeDef *i2s)
```

Comment:
Why is this function not in the source file?

Reply:
We're already doing this in spi driver for function which are only providing hw abstraction

Reply:
Improving driver file clarity. I agree that in current case, for one function, this might appear an excessive measure.
For spi driver case, there are lot of functions abstracted so it really made sense (this is all subjective, I agree).
Reusing the same scheme is for coherency sake across STM32 drivers.